### ISSUE TITLE
Link Checker: Understand and fix Notion false positives

### ISSUE THREAD

**Issue by janpio** [prisma/pris.ly#59]
Some Notion links consistently report 404 in the Link Checker, although they are manually reachabel:

```
[✖] https://www.notion.so/prismaio/Database-Workflows-API-Development-with-Prisma-We-Are-Developers-22-0c3074d901ec4bdf84fdc82056cbd263 → Status: 404
[✖] https://www.notion.so/prismaio/Prisma-Roadmap-50766227b779464ab98899accb98295f → Status: 404
[✖] https://www.notion.so/Prisma-Workshop-dada8b3c8aa142d5b0566932e1cdb058 → Status: 404
...
[✖] https://www.notion.so/prismaio/A-Practical-Introduction-to-Prisma-Prisma-Day-22-13988882ddf64a10a10b31122ba5e97d → Status: 404
[✖] https://prismaio.notion.site/Prisma-Free-and-Open-Source-Software-Fund-FOSS-Fund-061d28a663464cd6bf39b34ca73f29ab → Status: 404
```

We should understand that and optimally fix those so we can potentially use the GitHub Action to report actually broken links to Slack.

**Comment by janpio** [comment]
@Jolg42 Can you summarize what you found and close this? Thanks.

**Comment by Jolg42** [comment]
It seems that all links starting with `https://www.notion.so/prismaio/` are 404 because they require the computer who visits them to be logged in and have access to Prisma's Notion.
Opening these links in Safari (which is not logged in to Notion) on my computer showed the redirection to `https://prismaio.notion.site/...`

Besides that there were 2 links broken, see https://github.com/prisma/pris.ly/pull/75

Now tests are passing!

**Comment by Jolg42** [comment]
Note: This unblocks https://github.com/prisma/pris.ly/issues/61 (we needed reliable passing tests)
